olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -1121,25 +1121,25 @@ spec:
                       - name: "IMAGE_KN_PLUGIN_FUNC_PYTHON_39"
                         value: "registry.access.redhat.com/ubi8/python-39@sha256:27e795fd6b1b77de70d1dc73a65e4c790650748a9cfda138fdbd194b3d6eea3d"
                       - name: "KAFKA_IMAGE_kafka-broker-receiver__kafka-broker-receiver"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-receiver-rhel8@sha256:8bae52be788bdcb0203216b3f177999463cd9b67c657b56b925c4e77351bce29"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-receiver-rhel8@sha256:157869d1d86da18082998dcf6dbca8e76ef752010091e52e4508d97fc8632af9"
                       - name: "KAFKA_IMAGE_kafka-broker-dispatcher__kafka-broker-dispatcher"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-dispatcher-rhel8@sha256:0ed3dc06b836545e7a7d8947c862c3fa762b9146d5cf5c48d565e4bfd9b0ee6c"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-dispatcher-rhel8@sha256:a811fb24e9e140f817be4373855a60570893cf7f3656f5eb0760f444f4d23e48"
                       - name: "KAFKA_IMAGE_kafka-channel-receiver__kafka-channel-receiver"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-receiver-rhel8@sha256:8bae52be788bdcb0203216b3f177999463cd9b67c657b56b925c4e77351bce29"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-receiver-rhel8@sha256:157869d1d86da18082998dcf6dbca8e76ef752010091e52e4508d97fc8632af9"
                       - name: "KAFKA_IMAGE_kafka-channel-dispatcher__kafka-channel-dispatcher"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-dispatcher-rhel8@sha256:0ed3dc06b836545e7a7d8947c862c3fa762b9146d5cf5c48d565e4bfd9b0ee6c"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-dispatcher-rhel8@sha256:a811fb24e9e140f817be4373855a60570893cf7f3656f5eb0760f444f4d23e48"
                       - name: "KAFKA_IMAGE_kafka-controller__controller"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-kafka-controller-rhel8@sha256:41e347491101b0baa30f377cd7424993d1dc6ac061216353ddce0706fb0013cd"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-ekb-kafka-controller-rhel8@"
```

Comment:
seems wrong
/hold

Reply:
`time="2025-03-27T12:29:07Z" level=fatal msg="Error parsing image name \"docker://quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-135/kn-ekb-kafka-controller:latest\": reading manifest latest in quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-135/kn-ekb-kafka-controller: received unexpected HTTP status: 502 Bad Gateway"`

https://github.com/openshift-knative/serverless-operator/actions/runs/14106453614/job/39514035224

Reply:
Fixed again
/unhold